components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -3034,32 +3034,30 @@ ledger::PendingContributionInfoList PendingContributionsOnFileTaskRunner(
 }
 
 PendingContributionInfo PendingContributionLedgerToRewards(
-    const ledger::PendingContributionInfo& contribution) {
+    ledger::PendingContributionInfoPtr contribution) {
   PendingContributionInfo info;
-  info.publisher_key = contribution.publisher_key;
-  info.category = contribution.category;
-  info.verified = contribution.verified;
-  info.name = contribution.name;
-  info.url = contribution.url;
-  info.provider = contribution.provider;
-  info.favicon_url = contribution.favicon_url;
-  info.amount = contribution.amount;
-  info.added_date = contribution.added_date;
-  info.viewing_id = contribution.viewing_id;
-  info.expiration_date = contribution.expiration_date;
+  info.publisher_key = contribution->publisher_key;
+  info.category = contribution->category;
+  info.verified = contribution->verified;
+  info.name = contribution->name;
+  info.url = contribution->url;
+  info.provider = contribution->provider;
+  info.favicon_url = contribution->favicon_url;
+  info.amount = contribution->amount;
+  info.added_date = contribution->added_date;
+  info.viewing_id = contribution->viewing_id;
+  info.expiration_date = contribution->expiration_date;
   return info;
 }
 
 void RewardsServiceImpl::OnGetPendingContributionsUI(
     GetPendingContributionsCallback callback,
-    const std::vector<std::string>& json_list) {
+    ledger::PendingContributionInfoList list) {
```

Comment:
`list` should be `const`

Reply:
this one can't be const as then you get this error

```
../../brave/components/brave_rewards/browser/rewards_service_impl.cc:3060:44: error: call to deleted constructor of 'ledger::PendingContributionInfoPtr' (aka 'StructPtr<ledger::mojom::PendingContributionInfo>')
        PendingContributionLedgerToRewards(std::move(item));
                                           ^~~~~~~~~~~~~~~
../../mojo/public/cpp/bindings/struct_ptr.h:114:28: note: 'StructPtr' has been explicitly marked deleted here
  DISALLOW_COPY_AND_ASSIGN(StructPtr);
                           ^
../../brave/components/brave_rewards/browser/rewards_service_impl.cc:3037:46: note: passing argument to parameter 'contribution' here
    const ledger::PendingContributionInfoPtr contribution) {
                                             ^
1 error generated.
```

Reply:
Yeap apologies did not expand function